docs/pages/web/link.js

```diff
@@ -17,8 +17,8 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         description={generatedDocGen?.description}
         defaultCode={`
 <Text inline>
-  Find tips and best practices on the
-  <Link href="https://business.pinterest.com/" inline> Pinterest Business Site </Link>
+To see how you can grow your business, visit
+  <Link href="https://business.pinterest.com/advertise" inline externalLinkIcon="default" target="blank">Pinterest Ads</Link>
```

Comment:
technically, Pinterest links dont have the visit icon, just non-Pinterest urls.

Reply:
<img width="615" alt="Screen Shot 2022-09-27 at 9 09 27 PM" src="https://user-images.githubusercontent.com/10593890/192685673-839443f4-6fbd-4ef7-9ee1-2873d2d1eead.png">

Reply:
Hmm... Good point.  Not to get too esoteric here, but would a Pinterest marketing site be considered leaving Pinterest content?

Reply:
Seems like we're ok to move forward as it - https://pinterest.slack.com/archives/C014X9LTRCN/p1664388995735539

Reply:
Let's bring this to tomorrow's meeting as per thread